frontend/next.config.js

```diff
@@ -35,5 +35,15 @@ module.exports = withCSS({
     wampNearExplorerUrl:
       process.env.WAMP_NEAR_EXPLORER_URL || defaultWampNearExplorerUrl,
     googleAnalytics: process.env.NEAR_EXPLORER_GOOGLE_ANALYTICS
+  },
+  webpack: (config, { isServer }) => {
+    // Fixes npm packages that depend on `fs` module
+    if (!isServer) {
+      config.node = {
+        fs: "empty"
+      };
+    }
```

Comment:
Is this something wrong with nearlib? Do we have an issue there? Can we fix it there? /cc @janedegtiareva

Reply:
zeit/next.js#7755 the issue and method is here.

Reply:
Vlad and I talked about this and this definitely can fix in the nearlib, nearlib does not fit browser configuration and webpack maybe a good method to do with that

Reply:
Let's create a tracking issue on nearlib and put a link to the issue in a comment here.